src/plasma/plasma_manager.cc

```diff
@@ -43,6 +43,21 @@
 #include "state/task_table.h"
 #include "state/db_client_table.h"
 
+int handle_sigpipe(Status s, int fd) {
+  if (s.ok()) {
+    return 0;
+  }
+  if (errno == EPIPE || errno == EBADF || errno == ECONNRESET) {
+    ARROW_LOG(WARNING)
```

Comment:
I'd like to use ARROW_LOG like macros throughout the codebase eventually. We can copy logging.h over and rename it to RAY_LOG.